src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -93,7 +93,7 @@ private function handleConfig(ContainerBuilder $container, array $config, array
         $container->setParameter('api_platform.error_formats', $errorFormats);
         $container->setParameter('api_platform.eager_loading.enabled', $config['eager_loading']['enabled']);
         $container->setParameter('api_platform.eager_loading.max_joins', $config['eager_loading']['max_joins']);
-        $container->setParameter('api_platform.eager_loading.eager_only', $config['eager_loading']['eager_only']);
+        $container->setParameter('api_platform.eager_loading.force', $config['eager_loading']['force']);
```

Comment:
Why not force_eager here? I agree it's cleaner, but @dunglas talked about DX and having the same name for both configurations.